cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/index.html

```diff
@@ -46,7 +46,7 @@
 
 {% block content_main %}
     <h1>Bureau Structure</h1>
-    <p>Org chart last updated: August 3, 2016</p>
+    <p>Org chart last updated: July 26, 2016</p>
     <div class="org-chart">
```

Comment:
ok, updated in https://github.com/cfpb/cfgov-refresh/pull/2273/commits/1d811548ae8483d37b9a630958b59e559c6d5aa4